src/Avalonia.Base/Input/Gestures.cs

```diff
@@ -50,6 +50,10 @@ public static class Gestures
             RoutedEvent.Register<PullGestureEventArgs>(
                 "PullGesture", RoutingStrategies.Bubble, typeof(Gestures));
 
+        public static readonly RoutedEvent<HoldGestureEventArgs> HoldGestureEvent =
+            RoutedEvent.Register<HoldGestureEventArgs>(
+                "HoldGesture", RoutingStrategies.Bubble, typeof(Gestures));
```

Comment:
It's HoldingEvent as per https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.holding?view=winrt-22621

Reply:
Similarly with other events - Tapped, DoubleTapped.
On the other hand, `PullEvent` and `ScrollEvent` by itself are more confusing.